docs/using/examples/api-examples.rst

```diff
@@ -0,0 +1,14 @@
+:tocdepth: 2
+
+====================
+Flocker API Examples
```

Comment:
I think your concerns about adding these examples without automated testing needs to be considered by Roger and Michael, as these docs were requested for Dockercon, and Kai has put a lot of effort in to bring these in with manual tests. I'll message them directly.